### PR TITLE
Moved MemorySize and removed TryUtils. Closes #729.

### LINKED ISSUE #729: Move MemorySize from JES to backend project. Take a look to TryUtil as well.
Move MemorySize from JES to backend project

### PR DESCRIPTION
**Pull request by francares**

### CODE REVIEW

**Comment by francares** [comment]
Reviewers for this one: @geoffjentry and @scottfrazer

File: backend/src/main/scala/cromwell/backend/model/MemorySize.scala [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package cromwell.backend.validation
+package cromwell.backend.model
```
Comment (geoffjentry) [inline backend/src/main/scala/cromwell/backend/model/MemorySize.scala]: model?

**Comment by geoffjentry** [comment]
👍

File: backend/src/main/scala/cromwell/backend/model/MemorySize.scala (hunk 1 above)
Comment (francares) [inline backend/src/main/scala/cromwell/backend/model/MemorySize.scala]: Model package is commonly use in most of programming languages to put all VOs related to that component. The idea comes from MVC. 
i.e.: https://github.com/broadinstitute/rawls/tree/develop/src/main/scala/org/broadinstitute/dsde/rawls/model
https://github.com/broadinstitute/firecloud-orchestration/tree/develop/src/main/scala/org/broadinstitute/dsde/firecloud/model

Comment (geoffjentry) [inline backend/src/main/scala/cromwell/backend/model/MemorySize.scala]: I'm aware of what MVC is. I was asking why you think we're doing MVC here and what makes this an M

Comment (francares) [inline backend/src/main/scala/cromwell/backend/model/MemorySize.scala]: We are not doing MVC. What I'm trying to say is that is common to put all VOs that represents the model for that component together as I shown in the examples and it's common as well to create a folder called "model" even if you don't implemented a MVC or MVCS or MVP pattern.

Comment (geoffjentry) [inline backend/src/main/scala/cromwell/backend/model/MemorySize.scala]: I would rather keep things with their logical topics and not in one grab bag of stuff

**Comment by scottfrazer** [comment]
👍

**Comment by cjllanwarne** [comment]
@geoffjentry Just to confirm - was your comment above about `model` considered `Action Required` or `Thinking out Loud`?

**Comment by geoffjentry** [comment]
@cjllanwarne I'm against that change.

**Comment by geoffjentry** [comment]
@cjllanwarne Or at the very least I'm against the change here/now without a larger discussion of the topic.

**Comment by francares** [comment]
@geoffjentry @cjllanwarne I removed model package to avoid larger discussions on the topic.

**Pull request merged**